src/js/components/MultiSelect/stories/Children.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
```

Comment:
can we maybe change one of these stories to a typescript file?

Reply:
Yes, changed the Children story to be a typescript story